core/get_modules.py

```diff
@@ -53,7 +53,7 @@ def load_all_modules():
     # structure of module name: module_name = lib/modules/(category_name/module_name)/__init.py
     # example: module_name = lib/modules/(ftp/weak_password)/__init.py = ftp/weak_password
     module_names = []
-    for module in glob(os.path.dirname(inspect.getfile(lib)) + '/modules/*/*/__init__.py'):
+    for module in glob(os.path.dirname(inspect.getfile(modules)) + '/*/*/__init__.py'):
```

Comment:
maybe put the module_directory to a separate variable

Reply:
Pushed an update for this.